hledger-lib/Hledger/Reports/PostingsReport.hs

```diff
@@ -61,15 +65,44 @@ instance HasAmounts PostingsReportItem where
 -- with a tuple.
 type SummaryPosting = (Posting, Period)
 
+-- Possible value expressions taken by the --sort command
+-- Each of these takes a bool, which shows if it has been inverted
+-- (True -> has been inverted, reverse the order)
+data ValueExp
+    = Date' Bool
+    | Account' Bool
+    | Amount' Bool
+    deriving (Show)
+type SortSpec = [ValueExp]
```

Comment:
Naming: consider not using "value expressions" here. That specific phrase is used in Ledger's manual for Ledger's built in programming language, and in a number of docs I say that Ledger value expressions are something hledger doesn't support; and we'll never support value expressions exactly like Ledger's.

Reply:
Makes sense! How about `SortField` or something more directly descriptive then?